encoding/src/maccommands.rs

```diff
@@ -521,6 +521,23 @@ impl<const N: usize> ChannelMask<N> {
         self.0[index] = value;
     }
 
+
+    /// Enable or disable a specific channel. Recall that LoRaWAN channel numbers start indexing
+    /// at zero.
+    ///
+    /// Improper use of this method could lead to out of bounds panic during runtime!
+    pub fn set_channel(&mut self, channel: usize, set: bool) {
+        let index = channel / 8;
```

Comment:
nit(optional): for consistency with`channel_enabled`, I would prefer to have bit operations here.

Reply:
Good point! Fixed